pkg/lvm/lvm_util.go

```diff
@@ -868,13 +870,46 @@ func decodeLvsJSON(raw []byte) ([]LogicalVolume, error) {
 	return lvs, nil
 }
 
-func ListLVMLogicalVolume() ([]LogicalVolume, error) {
+// Change the attributes of an LV. Let the caller provide the args string
+// since we won't know what attrs and to what values need changing.
+func ChangeLVMLogicalVolumeAttrs(lvname string, args []string) error {
+	cmd := LVChange
+
+	// append lvname to args
+	args = append([]string{lvname}, args...)
+
+	// Are we running privileged or not?
+	if os.Geteuid() != 0 {
+		args = append([]string{cmd}, args...)
+		cmd = SUDO
+	}
+
+	_, _, err := RunCommandSplit(cmd, args...)
+	if err != nil {
+		klog.Errorf("lvm: error while running command %s %v: %v", LVChange, args, err)
+		return err
+	}
+
+	return nil
+}
+
+// Get all, or a particular LVM LV. If lvname is nil, all LVs are listed, otherwise the requested one.
+func ListLVMLogicalVolume(lvname string) ([]LogicalVolume, error) {
```

Comment:
I'd suggest using different functions rather than overload and having to pass empty string as you have above

Reply:
already updated diff in PR 308.